jscomp/others/js_option.ml

```diff
@@ -59,5 +59,27 @@ let andThen f x =
   | None -> None 
   | Some x -> f x [@bs]
 
+let map f x =
+  match x with
+  | None -> None
+  | Some x -> Some (f x [@bs])
 
+let orValue a x =
+  match x with
+  | None -> a
+  | Some x -> x
 
+let filter f x =
+  match x with
+  | None -> None
```

Comment:
cool, lets add `filter` then.
@OvermindDL1 there is no option module in ocaml stdlib so far as I know..